plugins/linux-v4l2/v4l2-output.c

```diff
@@ -175,19 +171,40 @@ static bool try_connect(void *data, int device)
 static bool virtualcam_start(void *data)
 {
 	struct virtualcam_data *vcam = (struct virtualcam_data *)data;
+	struct dirent **list;
+	bool success = false;
+	int n;
 
 	if (!loopback_module_loaded()) {
 		if (loopback_module_load() != 0)
 			return false;
 	}
 
-	for (int i = 0; i < MAX_DEVICES; i++) {
-		if (!try_connect(vcam, i))
+	n = scandir("/dev", &list, NULL, versionsort);
```

Comment:
Trying to connect to every device is not appropriate, add a filter to restrict this list to video devices.

Reply:
I see you have the filter below consider moving it in here.

Reply:
Thanks for the feedback. Should be addressed by the latest push.